app/controllers/claims/support/users_controller.rb

```diff
@@ -5,9 +5,48 @@ def index
     @users = @school.users
   end
 
+  def new
+    @user = Claims::User.new
+  end
+
+  def check
+    if user.valid?
+      @user = user.decorate
+    else
+      render :new
+    end
+  end
+
+  def create
+    if user.save
+      attach_user_to_school
+      send_invite_email_to_user
+      redirect_to claims_support_school_users_path(@school)
+      flash[:success] = "User added"
```

Comment:
The logic to save a user, attach it to a school, and sending an email feels like it belongs to a service like `UserInviteService.call` I think it's making it more reusable and testable